docker-compose.yml

```diff
@@ -1,6 +1,7 @@
 version: '3.8'
 services:
   pl:
+    platform: linux/amd64
```

Comment:
This is unrelated. Was it added by mistake?

Reply:
Hey yes sorry I added this when I was trying to fix my docker issues, just wanted to comment so I know this issue has been resolved :) Thanks for these comments!